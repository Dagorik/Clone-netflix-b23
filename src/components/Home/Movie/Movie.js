import React, {Component} from 'react'
import './movie.css'
class Movie extends Component{
    render(){
        return (
            <div className="col-md">
                <div className="card Movie__card">
                    <img className="card-img-top" src="https://upload.wikimedia.org/wikipedia/en/thumb/f/fe/Robo2poster.png/220px-Robo2poster.png" alt="Card image cap"/>
                    <div className="card-body">
                        <h5 className="card-title">Card title</h5>
                        <a className="btn btn-primary">Ver</a>
                    </div>
                </div>
            </div>
        )
    }
}

export default Movie;