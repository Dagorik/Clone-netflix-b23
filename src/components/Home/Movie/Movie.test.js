import React from 'react';
import {shallow} from "enzyme";
import Movie from './Movie';


describe("Movie",() => {
    const movieObject = {
        "poster":"www.google.com",
        "title":"La pelicula test",
        "id":"AHDUHSDUHQUWDUQDWU"
    }

    const component = shallow(<Movie title={movieObject.title} 
                    poster={movieObject.poster}
                    id={movieObject.id}
         />)

    it("Check if props passed correctly",() => {
       

        expect(component.find(".card-title").text()).toBe(movieObject.title)


    })

    it("Check movie renders fine",() =>{
        expect(component).toMatchSnapshot()
    })
})