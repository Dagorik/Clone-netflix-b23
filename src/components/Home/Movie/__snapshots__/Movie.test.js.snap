// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie Check movie renders fine 1`] = `
<div
  className="col-md d-flex align-items-stretch mt-5"
>
  <div
    className="card Movie__card"
  >
    <img
      alt="Card image cap"
      className="card-img-top"
      src="www.google.com"
    />
    <div
      className="card-body"
    >
      <h5
        className="card-title"
      >
        La pelicula test
      </h5>
      <Link
        replace={false}
        to="/movie/AHDUHSDUHQUWDUQDWU"
      >
        VER
      </Link>
    </div>
  </div>
</div>
`;
